apps/src/sites/studio/pages/levelbuilder_applab.js

```diff
@@ -27,18 +27,17 @@ $(document).ready(function () {
     "buzzer.playNotes": null,
     "buzzer.playSong": null,
     "accelerometer.getOrientation": null,
-    "accelerometer.start": null,
     "accelerometer.getAcceleration": null,
     "isPressed": null,
     "holdtime": null,
-    "soundSensor.start": null,
     "soundSensor.value": null,
     "soundSensor.getAveragedValue": null,
     "soundSensor.setScale": null,
-    "lightSensor.start": null,
+    "soundSensor.threshold": null,
     "lightSensor.value": null,
     "lightSensor.getAveragedValue": null,
     "lightSensor.setScale": null,
+    "lightSensor.threshold": null,
```

Comment:
Don't change this now, but we should really consolidate this list of blocks into the `lib/kits/maker` component, maybe even generate it dynamically from the maker dropletConfnig.